package.json

```diff
@@ -36,6 +36,7 @@
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
     "redux-thunk": "^2.0.1",
+    "sparql-client-2": "^0.6.0",
```

Comment:
Looks like sparql-client-2 is a fork and improvement over sparql-client. Compare:
https://github.com/thomasfr/node-sparql-client
with
https://github.com/eddieantonio/node-sparql-client (client-2)